### DISCUSSION TITLE
flinkcdc抽取ob for oracle的nvarchar字段乱码

### DISCUSSION THREAD

**Discussion by wanyujichun** [oceanbase/flink-connector-oceanbase#22] [Q&A] [answered]
![Uploading 截屏2023-05-06 15.19.19.png…]()

**Comment by whhe** [comment, marked as answer]
截图我这边看不到。目前 flink cdc 的 release 版本应该还是不支持 oceanbase 的 oracle 租户的，想问下你是怎么用的？直接用现有的代码接了增量吗？nvarchar 在这里映射的数据类型是什么？

**Reply by wanyujichun** [reply]
您好：
1. 我们使用的是flink-connector-oceanbase-cdc-2.3.0.jar
2.目前参照官方的说明，可以直接读取oracle租户的增量数据，就是nvarchar遇到了问题，其他目前正常
3.我们自己debug下看数据从obproxy出来，在反序列化之前已经是乱码
![WechatIMG665](https://user-images.githubusercontent.com/8925285/236613635-f2851e32-ff68-4f63-a83b-309bfe95bb95.jpeg)

**Reply by whhe** [reply]
关联信息：
- https://ask.oceanbase.com/t/topic/35603320
- https://github.com/ververica/flink-cdc-connectors/issues/2114
